audit/projects/k8s-artifacts-prod-bak/buckets/asia.artifacts.k8s-artifacts-prod-bak.appspot.com/metadata.txt

```diff
@@ -14,7 +14,7 @@ gs://asia.artifacts.k8s-artifacts-prod-bak.appspot.com/ :
 	Time updated:			Fri, 10 Sep 2021 21:03:19 GMT
 	Metageneration:			36
 	Bucket Policy Only enabled:	True
-	Public access prevention:	unspecified
+	Public access prevention:	inherited
```

Comment:
Looks like a rollout of https://cloud.google.com/storage/docs/public-access-prevention.